src/Symfony/Component/ObjectMapper/.github/workflows/check-subtree-split.yml

```
@@ -0,0 +1,37 @@
+name: Check subtree split
+
+on:
+  pull_request_target:
+
+jobs:
+  close-pull-request:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Close pull request
+      uses: actions/github-script@v6
+      with:
+        script: |
+          if (context.repo.owner === "symfony") {
+            github.rest.issues.createComment({
+              owner: "symfony",
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              body: `
+          Thanks for your Pull Request! We love contributions.
+
+          However, you should instead open your PR on the main repository:
+          https://github.com/symfony/symfony
+
+          This repository is what we call a "subtree split": a read-only subset of that main repository.
+          We're looking forward to your PR there!
+          `
+            });
+
+            github.rest.pulls.update({
+              owner: "symfony",
+              repo: context.repo.repo,
+              pull_number: context.issue.number,
+              state: "closed"
+            });
+          }
```

This is an old version of the action that was no working. The new version is already on every other subtree splits:

```suggestion
name: Close Pull Request

on:
  pull_request_target:
    types: [opened]

jobs:
  run:
    runs-on: ubuntu-latest
    steps:
    - uses: superbrothers/close-pull-request@v3
      with:
        comment: |
          Thanks for your Pull Request! We love contributions.

          However, you should instead open your PR on the main repository:
          https://github.com/symfony/symfony

          This repository is what we call a "subtree split": a read-only subset of that main repository.
          We're looking forward to your PR there!
```

Note that you can run `.github/sync-packages.php` to update it.

---

thanks!